teuthology/provision.py

```diff
@@ -195,6 +200,168 @@ def __del__(self):
         self.remove_config()
 
 
+class ProvisionOpenStack(OpenStack):
```

Comment:
create() and destroy(), definitely. The build_config / remove_config are not shared, I think.